Automattic/wp-calypso
# Patterns: category page

Related to https://github.com/Automattic/dotcom-forge/issues/5849

## Proposed Changes
In the context of work related to the Patterns pages, this pull request includes the addition of markup for the `/patterns/:category` page.

## Testing Instructions
1. Open http://calypso.localhost:3000/patterns
2. Click on any pattern (e.g. `About`)
3. Asert that you are landed to `http://calypso.localhost:3000/patterns/about`
4. Assert that `About` category is active<br /> ![Screenshot 2024-03-06 at 15 40 02](https://github.com/Automattic/wp-calypso/assets/5598437/d8d87738-9e0d-4dfd-a995-5e52e20edfe3)
5. Assert that in general you see the next page  <br /> ![Screenshot 2024-03-06 at 15 41 02](https://github.com/Automattic/wp-calypso/assets/5598437/740c8346-d2c2-4c66-853c-3d1e42ab0607)
6. Assrt that navigation works well - click on items and scrolling left/right with clicks on arrows, also it's possible to scroll it with mouse-wheel
7. Assert that mobile version looks good and navigation doesn't have arrows but it's possible to scroll it with mouse-wheel <br /> ![Screenshot 2024-03-06 at 15 52 30](https://github.com/Automattic/wp-calypso/assets/5598437/e526c8d6-6e41-4223-8f55-6ed454d4212c)
